platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/builder/PlatformBuilder.java

```diff
@@ -557,6 +575,10 @@ public PlatformComponentBuilder buildComponentBuilder() {
             throw new IllegalStateException("The current address book of the initial state is null.");
         }
 
+        if (roster == null) {
+            roster = createRoster(boostrapAddressBook);
```

Comment:
The `boostrapAddressBook` is never used after this line. This means that the AddressBook instance that this variable describes is not useful. It's the `addressBook` defined at line 572 above that is used below. We need a Roster instance of that `addressBook` here.